src/app/public/styles/docs-tools.scss

```diff
@@ -48,4 +48,9 @@
       margin-top: 0 !important;
     }
   }
-}
\ No newline at end of file
+}
+
+.sky-docs-text-block {
```

Comment:
This global CSS class is a migration of the `skyux2-docs` class, `.sky-text-block` per Adam's request.